newIDE/app/src/UI/TextField.js

```diff
@@ -82,7 +82,7 @@ type Props = {|
   margin?: 'none' | 'dense',
   fullWidth?: boolean,
   style?: {|
-    fontSize?: 14 | 18,
+    fontSize?: 14 | 18 | string,
```

Comment:
Should this be a number?

Reply:
Well, it looks a bit awkward like this, not really happy with that.
It seems that TextFields can have only 2 values for their fontsize, which seems ok to me, for consistence's sake.
But in this case, I wanted the TextField to have an adaptable fontsize with `0.8em` for instance. Hence this line

Reply:
Ok! You could almost list the valid values? The idea was indeed to enforce consistency by having this kind of discussion "You're adding yet-another fontSize? You sure?" :) Maybe it's not too bad to list the values we're using, which is some kind of "living documentation" of the consistency across the app.

Reply:
Oh yeah, great idea!